frontend/pages/initiative/[id]/internal.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+const pactaClient = await usePACTA()
+const { fromParams } = useURLParams()
+
+const id = presentOrCheckURL(fromParams('id'))
+const prefix = `initiative/${id}/invitations`
+const [
+  { data: initiative },
+] = await Promise.all([
+  useSimpleAsyncData(`${prefix}.getInitiative`, () => pactaClient.findInitiativeById(id)),
+  useSimpleAsyncData(`${prefix}.getInvitations`, () => pactaClient.listInitiativeInvitations(id)),
+  useSimpleAsyncData(`${prefix}.getRelationships`, () => pactaClient.listInitiativeUserRelationshipsByInitiative(id)),
+])
```

Comment:
Any reason this can't just be a hidden/not-hidden section on the `/initiative/[id]` page? Esp since it appears to need the exact same set of data.

Reply:
^^